generator/sig_readme.tmpl

```diff
@@ -38,6 +38,16 @@ subprojects, and resolve cross-subproject technical issues and decisions.
 * {{.Name}} (**[@{{.GitHub}}](https://github.com/{{.GitHub}})**){{if .Company}}, {{.Company}}{{end}}
 {{- end }}
 {{- end }}
+{{- if .Leadership.ProgramManagers }}
+
+### Program Managers
+The Program Managers of the SIG provide strategic guidance to optimize delivery
+of SIG objectives, as well as facilitate collaboration and communication on
+cross-project initiatives.
```

Comment:
@LappleApple -- this is something I word-souped. Do you want to put your own flair on this description?

Reply:
The flair has been put!